one_fm/overrides/leave_application.py

```diff
@@ -22,7 +25,26 @@ def validate(self):
         if frappe.db.get_value("Leave Type", self.leave_type, "is_optional_leave"):
             self.validate_optional_leave()
         self.validate_applicable_after()
-
+        
+    def assign_to_leave_approver(self):
+        #This function is meant to create a TODO for the leave approver
+        if self.name:
+            existing_assignment = frappe.get_all("ToDO",{'allocated_to':self.leave_approver,'reference_name':self.name})
+            if not existing_assignment:
+                try:
+                    todo_doc = frappe.get_doc({
+                        'doctype':'ToDo',
+                        'allocated_to':self.leave_approver,
+                        'reference_type':'Leave Application',
+                        'description':f'Please note that leave application {self.name} is awaiting for your approval',
+                        'reference_name':self.name
+                    })
+                    todo_doc.insert()
+                    todo_doc.save()
+                except:
+                    
+                    frappe.log_error(frappe.get_traceback(),"Error assigning to User")
```

Comment:
`from frappe.desk.form.assign_to import add` We can make use of this implementation for assignment

Reply:
@pjamsheer Thanks alot. I will quickly make the changes